src/assets/i18n/en.json5

```diff
@@ -1676,6 +1684,28 @@
 
 
 
+  "item.listelement.badge": "Item",
+
+  "item.page.description": "Description",
+
+  "item.page.edit": "Edit this item",
+
+  "item.page.journal-issn": "Journal ISSN",
+
+  "item.page.journal-title": "Journal Title",
+
+  "item.page.publisher": "Publisher",
+
+  "item.page.titleprefix": "Item: ",
```

Comment:
I'd rather we leave off the Title prefix for a default/untyped "Item", just like we do so for Communities/Collections.  Currently, this looks a bit odd on Item pages to have "Item:" before every title.

Reply:
Agreed